tests/test_sv.py

```diff
@@ -117,8 +120,28 @@ def test_validation_no_errors(self):
                 "CONNECTION_TYPE": ["3to5", "5to5"],
                 "DNA_SUPPORT": ["Yes", "No"],
                 "RNA_Support": ["Yes", "No"],
+                "SITE1_CHROMOSOME": [1, 22],
+                "SITE2_CHROMOSOME": ["X", "2"],
             }
         )
         error, warning = self.sv_cls._validate(sv_df)
         assert error == ""
         assert warning == ""
+
+    def test_validation__validate_chromosome_is_called(self):
+        """Tests that _validate_chromosome is called twice
+        for the two chromosome columns that sv files may have"""
+        sv_df = pd.DataFrame(
+            {
+                "sample_id": ["GENIE-SAGE-ID1-1", "GENIE-SAGE-ID2-1"],
+                "SITE1_CHROMOSOME": [1, 22],
+            }
+        )
+        with mock.patch(
```

Comment:
Nit: I know this wasn't documented, but for consistency across the package, this is how mocking is done. (I also find it a little easier)

```
from unittest.mock import patch
patch.object(validate, "_validate_chromosome". return_value=("" ,"")
```